### ISSUE TITLE
Feature Request: Apple TV Support

### ISSUE THREAD

**Issue by unlive74** [amnezia-vpn/amnezia-client#911]
Guys, thank you for your work! 

I think it would be easy to add AmneziaVPN to the appstore for Apple TV. This is relevant in connection with the increasing blocking of YouTube and sometimes you just want to change the region.

**Comment by debug45** [comment]
Yes, the Apple TV app is really necessary. Especially because it is really easy to upgrade the iOS app for tvOS.

**Comment by r4hx** [comment]
Can we help you with the release of the version for apple tv?

**Comment by pokamest** [comment]
@r4hx Please contact Telegram @amnezia_service if you want to contribute

**Comment by mrtrofimov** [comment]
Any news about the Apple TV app? It is very desired version. Please take it into your plans.

**Comment by niiiksh** [comment]
Any news for April 2025?

**Comment by WinManMC** [comment]
Any news for Jul 2025?

**Comment by xahon** [comment]
Any news mid Jul 2025?

**Comment by juneaucross** [comment]
Hey, guys! You might not need exactly Amnezia app on your TV. For me, the solution is [https://apps.apple.com/app/id932747118](https://apps.apple.com/app/id932747118) + Amnezia’s SOCKS5 proxy server (which you can set up in less than a minute in the phone app).

_Delete if offtopic or breaking any rules._

**Comment by krasivan** [comment]
@juneaucross you're quite right! Let me just provide a few details. In Amnezia app go to the _Settings_ > _Servers_ > _*Your server name*_ > _Services_ tab  and click _SOCKS5 proxy server_. Then download the app from the OP on iPhone and create a new connection. Finally download it to your Apple TV and sync the app on your Apple TV with the iPhone app. Enjoy!

**Comment by niiiksh** [comment]
> @juneaucross you're quite right! Let me just provide a few details. In Amnezia app go to the _Settings_ > _Servers_ > _*Your server name*_ > _Services_ tab  and click _SOCKS5 proxy server_. Then download the app from the OP on iPhone and create a new connection. Finally download it to your Apple TV and sync the app on your Apple TV with the iPhone app. Enjoy! 

What is Amnezia's app? I'm using AmneziaWG and there is no option for proxy server. The topic is about AmneziaWG app for Apple TV.

**Comment by kmishukov** [comment]
Any updates? December 2025
Ready to contribute.

upd. 

Wireguaed + AmneziaWG + UDP via Shadowrocket. Runs smoothly.

**Comment by niiiksh** [comment]
> Any updates? December 2025 Ready to contribute.

Try to setup Vless and use Happ for tvOS better

**Comment by AriAriPee** [comment]
The devs told me it might be ready by New Year’s, but it’s already February and there’s still no progress.![image](https://github.com/user-attachments/assets/466fab83-8166-4044-bbb6-a76fbd0da9bb)

**Comment by BossyBigBoss** [comment]
Any news regarding the subject?

**Comment by quberok** [comment]
Up

**Comment by xf072161** [comment]
giga up
